### PR TITLE
R1 to Add PT-BR automatic translation softwarereview_author.pt.Rmd

### PR DESCRIPTION
**Pull request by samuel-carleial**
First round of translation (R1)

### CODE REVIEW

**Comment by samuel-carleial** [comment]
Here I also have some comments and questions on the translation of terms.
@pedropark99  and @beatrizmilz see below:

software peer review -> "revisão de software por pares"
badges -> "crachá" ou "distintivo"? eu escolhi o ultimo
*Active*, *WIP* badges -> mantive o termo em italico
lifecycle badges -> distintivos tipo _lifecycle_
Packaging Guide -> guia de pacotes
GitHub Action -> termo mantido em inglês
our base Dockerfile -> nosso arquivo Dockerfile

Please let me know what do you think and if you have a better suggestion. Obrigado

**Comment by pedropark99** [comment]
Honestly, I think you did a very good job! I'm very ok with your choices 😉

File: softwarereview_author.pt.Rmd [hunk 1]
```diff
@@ -1,61 +1,62 @@
...
+- Considere também o tempo e o esforço necessários para responder às revisões: pense na sua disponibilidade ou na de seus colaboradores nas próximas semanas e meses após o envio da submissão. Observe que os revisores são voluntários e pedimos que você respeite o tempo e o esforço deles, respondendo de maneira oportuna e respeitosa.
+- Se você usa [distintivos do repostatus.org](https://www.repostatus.org/) (o que recomendamos), envie uma submissão quando você estiver pronto para receber um distintivo tipo _Active_ em vez de _WIP_. Da mesma forma, se você usa [distintivos tipo _lifecycle_](https://www.tidyverse.org/lifecycle/) o envio da submissão deverá ocorrer quando o pacote for _Stable_.
+- Para qualquer envio ou consulta de pré-submissão, o LEIAME do seu pacote deve fornecer informações suficientes sobre o pacote (objetivos, uso, pacotes semelhantes) para que os editores avaliem seu escopo sem precisar instalar o pacote. Melhor ainda, crie um website pkgdown para permitir uma avaliação mais detalhada da funcionalidade online.
```
Comment (pedropark99) [inline softwarereview_author.pt.Rmd:23]: I think keeping the term README, and not translating it makes more sense to me.

```suggestion
- Para qualquer envio ou consulta de pré-submissão, o README do seu pacote deve fornecer informações suficientes sobre o pacote (objetivos, uso, pacotes semelhantes) para que os editores avaliem seu escopo sem precisar instalar o pacote. Melhor ainda, crie um website pkgdown para permitir uma avaliação mais detalhada da funcionalidade online.
```

File: softwarereview_author.pt.Rmd [hunk 2]
```diff
@@ -1,61 +1,62 @@
...
+  - No estágio de envio da submissão, todas as principais funções devem ser estáveis o suficiente para serem totalmente documentadas e testadas; o LEIAME deve apresentar uma base segura para o pacote.
+  - Seu arquivo LEIAME deve assegurar-se em explicar a funcionalidade e os objetivos do seu pacote, presumindo que os leitores tenham pouco ou nenhum conhecimento do domínio. Todos os termos técnicos, inclusive as referências a outros softwares, devem ser esclarecidos.
```
Comment (pedropark99) [inline softwarereview_author.pt.Rmd:25]:
```suggestion
  - Seu arquivo README deve assegurar-se em explicar a funcionalidade e os objetivos do seu pacote, presumindo que os leitores tenham pouco ou nenhum conhecimento do domínio. Todos os termos técnicos, inclusive as referências a outros softwares, devem ser esclarecidos.
```

File: softwarereview_author.pt.Rmd [hunk 3]
```diff
@@ -1,61 +1,62 @@
...
+- Se você usa [distintivos do repostatus.org](https://www.repostatus.org/) (o que recomendamos), envie uma submissão quando você estiver pronto para receber um distintivo tipo _Active_ em vez de _WIP_. Da mesma forma, se você usa [distintivos tipo _lifecycle_](https://www.tidyverse.org/lifecycle/) o envio da submissão deverá ocorrer quando o pacote for _Stable_.
+- Para qualquer envio ou consulta de pré-submissão, o LEIAME do seu pacote deve fornecer informações suficientes sobre o pacote (objetivos, uso, pacotes semelhantes) para que os editores avaliem seu escopo sem precisar instalar o pacote. Melhor ainda, crie um website pkgdown para permitir uma avaliação mais detalhada da funcionalidade online.
+  - No estágio de envio da submissão, todas as principais funções devem ser estáveis o suficiente para serem totalmente documentadas e testadas; o LEIAME deve apresentar uma base segura para o pacote.
```
Comment (pedropark99) [inline softwarereview_author.pt.Rmd:24]:
```suggestion
  - No estágio de envio da submissão, todas as principais funções devem ser estáveis o suficiente para serem totalmente documentadas e testadas; o README deve apresentar uma base segura para o pacote.
```

File: softwarereview_author.pt.Rmd [hunk 4]
```diff
@@ -1,61 +1,62 @@
...
-- Se o seu pacote exigir dependências de sistema incomuns (consulte [*Guia de empacotamento*](#pkgdependencies)) para que nossa Ação do GitHub seja aprovada, envie um pull request adicionando-os ao [nosso Dockerfile básico](https://github.com/ropensci-review-tools/pkgcheck/blob/main/Dockerfile).
-- Se houver algum aspecto do `pkgcheck` que seu pacote não possa ser aprovado, explique os motivos no seu modelo de envio.
+## Planejando uma Submissão (ou uma Consulta de Pré-Submissão) {#planning-a-submission-or-a-pre-submission-enquiry}
+
+- Você pretende manter seu pacote por pelo menos 2 anos ou ser capaz de identificar um novo mantenedor?
```
Comment (pedropark99) [inline softwarereview_author.pt.Rmd:14]: Make it more gender neutral

```suggestion
- Você pretende manter o seu pacote por pelo menos 2 anos ou ser capaz de identificar um(a) novo(a) mantenedor(a)?
```

File: softwarereview_author.pt.Rmd [hunk 5]
```diff
@@ -1,61 +1,62 @@
...
+- Consulte nossas [políticas de uso](#policies) para ver se o seu pacote atende aos nossos critérios e se encaixa em nossa coleção, não se sobrepondo a outros pacotes.
```
Comment (pedropark99) [inline softwarereview_author.pt.Rmd:15]:
```suggestion
- Consulte nossas [políticas de uso](#policies) para ver se o seu pacote atende aos nossos critérios e se encaixa em nossa coleção, não se sobrepondo a outros pacotes já existentes.
```

File: softwarereview_author.pt.Rmd [hunk 6]
```diff
@@ -1,61 +1,62 @@
...
+  - Se você não tiver certeza de que um pacote atende aos nossos critérios, sinta-se à vontade para abrir um problem (_issue_) como uma consulta de pré-submissão para perguntar se o pacote é apropriado.
```
Comment (pedropark99) [inline softwarereview_author.pt.Rmd:16]:
```suggestion
  - Se você não tiver certeza de que um pacote atende aos nossos critérios, sinta-se à vontade para abrir um  _issue_ no GitHub como uma consulta de pré-submissão para perguntar se o pacote é apropriado.
```

File: softwarereview_author.pt.Rmd [hunk 7]
```diff
@@ -1,61 +1,62 @@
...
+  - [Exemplo de resposta a sobreposição](https://github.com/ropensci/software-review/issues/199#issuecomment-375358362). Também considere adicionar alguns pontos sobre pacotes semelhantes ao seu na sua [documentação do pacote](#docs-general).
+- Considere o melhor momento de desenvolvimento do seu pacote para enviar sua submissão. Seu pacote deve estar suficientemente maduro para que os revisores possam analisar todos os aspectos essenciais, mas tenha em mente que revisões podem resultar em grandes alterações.
+  - Sugerimos enfaticamente que você envie seu pacote para análise *antes de* publicá-lo no CRAN ou enviá-lo para publicação como artigo em um periódico. O _feedback_ da revisão pode resultar em grandes aprimoramentos e atualizações do seu pacote, incluindo renomeações e alterações de funções.
```
Comment (pedropark99) [inline softwarereview_author.pt.Rmd:19]:
```suggestion
  - Sugerimos enfaticamente que você envie seu pacote para análise *antes de* publicá-lo no CRAN ou antes de enviá-lo para publicação como artigo em um periódico. O _feedback_ da revisão pode resultar em grandes aprimoramentos e atualizações do seu pacote, incluindo renomeações e alterações de funções.
```

File: softwarereview_author.pt.Rmd [hunk 8]
```diff
@@ -1,61 +1,62 @@
...
+- Para qualquer envio ou consulta de pré-submissão, o LEIAME do seu pacote deve fornecer informações suficientes sobre o pacote (objetivos, uso, pacotes semelhantes) para que os editores avaliem seu escopo sem precisar instalar o pacote. Melhor ainda, crie um website pkgdown para permitir uma avaliação mais detalhada da funcionalidade online.
+  - No estágio de envio da submissão, todas as principais funções devem ser estáveis o suficiente para serem totalmente documentadas e testadas; o LEIAME deve apresentar uma base segura para o pacote.
+  - Seu arquivo LEIAME deve assegurar-se em explicar a funcionalidade e os objetivos do seu pacote, presumindo que os leitores tenham pouco ou nenhum conhecimento do domínio. Todos os termos técnicos, inclusive as referências a outros softwares, devem ser esclarecidos.
+- Seu pacote continuará a evoluir após a revisão. O capítulo sobre *Evolução do pacote* [fornece orientação sobre o tópico](#evolution).
```
Comment (pedropark99) [inline softwarereview_author.pt.Rmd:26]:
```suggestion
- Seu pacote continuará a evoluir após a revisão. O capítulo sobre *Evolução do pacote* [fornece mais orientações sobre este tópico](#evolution).
```

File: softwarereview_author.pt.Rmd [hunk 9]
```diff
@@ -1,61 +1,62 @@
...
+
+## Preparando para Submissão {#preparing-for-submission}
+
+- Leia e siga [nosso guia de estilo de pacotes](#building) e nosso [guia do revisor](#preparereview), para garantir que seu pacote atenda aos nossos critérios de estilo e qualidade.
+- Fique à vontade para fazer perguntas sobre o processo ou sobre seu pacote em específico no nosso [Fórum de discussão](https://discuss.ropensci.org).
+- Todas as submissões são verificadas automaticamente pelo nosso [pkgcheck](https://docs.ropensci.org/pkgcheck/) para garantir que os pacotes seguem nossas diretrizes. Espera-se que todos os autores tenham executado a [principal função do `pkgcheck`](https://docs.ropensci.org/pkgcheck/reference/pkgcheck.html) localmente para confirmar que o pacote está pronto para ser submetido. Como alternativa, uma maneira ainda mais fácil de garantir que um pacote está pronto para ser submetido é usando [a função `pkgcheck` do GitHub Action](https://github.com/ropensci-review-tools/pkgcheck-action), conforme descrito em [nossa postagem no blog](https://ropensci.org/blog/2022/02/01/pkgcheck-action/).
```
Comment (pedropark99) [inline softwarereview_author.pt.Rmd:32]:
```suggestion
- Todas as submissões são verificadas automaticamente pelo nosso [`pkgcheck`](https://docs.ropensci.org/pkgcheck/) para garantir que os pacotes seguem nossas diretrizes. Espera-se que todos os autores tenham executado a [principal função do `pkgcheck`](https://docs.ropensci.org/pkgcheck/reference/pkgcheck.html) localmente para confirmar que o pacote está pronto para ser submetido. Como alternativa, uma maneira ainda mais fácil de garantir que um pacote está pronto para ser submetido é usando [a função `pkgcheck` do GitHub Action](https://github.com/ropensci-review-tools/pkgcheck-action), conforme descrito em [nossa postagem no blog](https://ropensci.org/blog/2022/02/01/pkgcheck-action/).
```

File: softwarereview_author.pt.Rmd [hunk 10]
```diff
@@ -1,61 +1,62 @@
...
 
-## O processo de envio {#the-submission-process}
+## O Processo de Submissão {#the-submission-process}
 
-- O software é enviado para análise por [abrindo uma nova edição](https://github.com/ropensci/software-review/issues/new/choose) no repositório de análise de software e preenchendo o modelo.
-- O modelo começa com uma seção que inclui diversas variáveis no estilo HTML (`<!---variable--->`). Elas são usadas pelo nosso `ropensci-review-bot` e devem ser deixadas no lugar, com valores preenchidos entre os pontos de início e fim indicados, assim:
+- Um software é enviado a revisão (submetido) ao ser  [aberta uma nova _issue_](https://github.com/ropensci/software-review/issues/new/choose) no repositório de revisão do software, sendo preenchido o modelo sugerido.
```
Comment (pedropark99) [inline softwarereview_author.pt.Rmd:40]:
```suggestion
- Um software é enviado/submetido para revisão através da [abertura de uma nova _issue_](https://github.com/ropensci/software-review/issues/new/choose) no repositório de revisão do software, sendo preenchido o modelo sugerido.
```

**Review by pedropark99 — COMMENTED** [review]
Thank you very much @samuel-carleial for your contributions! This is my review as the second reviewer in this PR.

**Comment by yabellini** [comment]
Thank you, @pedropark99, for the second review! @samuel-carleial, when you have the time to check Pedro's suggestions, you can apply them by clicking the "commit suggestion" button. Ping me when you finish, and I will merge the PR.

File: softwarereview_author.pt.Rmd (hunk 1 above)
Comment (samuel-carleial) [reply softwarereview_author.pt.Rmd:23]: The problem is that it has been translated to 'LEIAME' in other parts of the website. For consistency, the final revision of the whole content would need to check this out. I will accept your suggestion, but keep the note for the future.

**Comment by samuel-carleial** [comment]
Its done @yabellini

**Pull request merged**